pystemd/run.py

```diff
@@ -229,6 +234,10 @@ def bus_factory():
                 or None,
             }
         )
+        if user is DynamicUser:
```

Comment:
personal nit, i like this comparison more like `issubclass(DynamicUser, DynamicUser) ` but i guess its does not really matter